deringenieur71/Adventures-in-LOCOMOTIVE-BASIC
# Correct typo in werewolfves_and_wanderer.bas

## Summary of change

Corrected spelling: `COSUB` -> `GOSUB`; causes syntax error when using shop (`I`) to try to make any purchase when you have insufficient money

## Test instructions

If necessary:

1. Play the game, use command `I` to open the inventory/shop
2. Spend all your money
3. Continue to try to spend money
4. See syntax error

Switch to this branch and repeat the test; don't see a syntax error.
